src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -598,6 +598,21 @@ public function testClosureProxy()
     }
 
     /**
+     * @requires PHP 7.0
+     */
+    public function testClosureProxyWithVoidReturnType()
+    {
+        $container = include self::$fixturesPath.'/containers/container35.php';
```

Comment:
please give this file to a meaningful name (we're changing policy here, numbering is conflict prone)